tests/stratified_geometry_test_cases.py

```diff
@@ -0,0 +1,63 @@
+"""Core parametrizer classes for Stratified Spaces.
+"""
+
+import geomstats.backend as gs
+import geomstats.tests
+from tests.conftest import TestCase
+
+
+@geomstats.tests.np_only
+class PointSetTestCase(TestCase):
+    def test_random_point_belongs(self, space_args, n_points):
+        # TODO: add belongs_atol?
+        # TODO: it is a copy of ManifoldTestCase
+        space = self._PointSet(*space_args)
+        random_point = space.random_point(n_points)
+        result = gs.all(space.belongs(random_point))
+        self.assertAllClose(result, True)
+
+    def test_belongs(self, space_args, points, expected):
+        space = self._PointSet(*space_args)
+        self.assertAllClose(space.belongs(points), expected)
+
+    def test_set_to_array(self, space_args, points, expected):
+        space = self._PointSet(*space_args)
+
+        self.assertAllClose(space.set_to_array(points), expected)
```

Comment:
We could (and should in my opinion) automate `test_set_to_array` using the `test_to_array` method from `PointTestCase`; as the expected output of `set_to_array` is just the method `to_array` from `Point` applied to each element, no?
Not sure if this is trivial to do, as one would need to draw the smoke_data from `test_to_array`. I think it is worth giving a thought.

Reply:
In the case of spider the outputs are different: `point.to_array()` only translates ray and R value to an array, whereas `point_set.set_to_array(point)` creates an array of zeros with R from point assigned to the ray position.

@annacalissano may add something here.